grp/classic.gi

```diff
@@ -1917,6 +1917,122 @@ InstallMethod( Omega,
     { filt, e, d, R } -> OmegaCons( filt, e, d, Size( R ) ) );
 
 
+#############################################################################
+##
+#F  WallForm( <form>, <m>, <fld> ) . . . . . . . . . . . compute the wall of <m>
+##
+# Input: <m> is an element of the orthogonal group which is given by the form <form>,
+#        <fld> is the finite field of the orthogonal group (in which <m> is defined)
+# Output: The Wall Form of <m> correpsponding to <form>
+# Computes the Wall Form of a matrix. The definition can be found in
+# [Taylor, page 163].
```

Comment:
```suggestion
##  Return the Wall form of <m>, where <m> is a matrix over <fld> which is orthogonal
##  with respect to the bilinear form <form>, also given as a matrix.
##  For the definition of Wall forms, see [Taylor, page 163].
```
The reference should be clearer, though. In fact I have no idea what this refers to, could you please clarify?